examples/GTest/foo.cpp

```diff
@@ -1,5 +0,0 @@
-#include <gtest/gtest.h>
```

Comment:
Ok I just read that the main function is not needed when linking to GTest::main. I will write something about that in the comments and change the content of useGTest.cpp back to the former content of foo.cpp